docs/application/native/guides/text-input/autofill.md

```diff
@@ -1,39 +1,42 @@
 # Autofill
 
 Autofill is a feature that allows you to fill commonly entered information, such as email, account, and address in a text input field.
-The autofill functions is used to retrieve previously entered data by the user or request to store data in an autofill service.
+The Autofill functions is used to retrieve previously entered data by the user or request to store data in an Autofill service.
 
 The main features of the Autofill API include:
 
-- [Autofill Data and Autofill Authentication Information](#autofill-data-and-autofill-authentication-information)
+- Autofill Data and Autofill Authentication Information
 
-  You can get the autofill service information, such as autofill service name, service message, and service logo image to use `autofill_auth_info_request()`.
-  Also, can get the presence of autofill data for each view in autofill service information to be received.
+  You can request to [get the autofill service information](#autofill-data-and-autofill-authentication-information), such as Autofill service name, service message, and service logo image. You can get the existing Autofill data for each view in the Autofill service information received.
 
-- [Data entered and Stored in Autofill Service](#data-entered-and-stored-in-autofill-service)
+- Data Entered and Stored in Autofill Service
 
-  You can retrieve the data previously entered by the user and stored in the Autofill service using the `autofill_fill_request` function.
+  You can [retrieve the data](#data-entered-and-stored-in-autofill-service) that was stored in the Autofill service.
 
-- [Storing Entered Data](#storing-entered-data)
+- Save Autofill Data
 
-  You can store the data entered by the user in Autofill service using the `autofill_commit()` function.
+  You can [store](#save-autofill-data) the data entered by the user in an Autofill service.
 
 ## Prerequisites
 
-1. To configure the autofill service:
+1. To configure the Autofill Service:
 
-   Autofill service must be configured, to configure, set autofill service by navigating to Settings > Language and input > Input Assistance > Autofill setting.
-   In some other environments, the Autofill service can be found within the system configuration
+   Autofill Service must be configured, to configure, set Autofill Service by navigating to **Settings > Language and input > Input Assistance > Autofill service**.
```

Comment:
Please verify:
Set Autofill Service by navigating to **Settings > Language and input > Input Assistance > Autofill service**.

Reply:
Fixed.